package.json

```diff
@@ -13,7 +13,7 @@
     "e2e:debug": "./e2e/start-and-run-suite debug",
     "e2e:dev": "./e2e/start-and-run-suite dev",
     "jest": "jest --notify --watch",
-    "lint": "eslint . --ext .js,.tsx,.ts --cache --ignore-path .gitignore --ignore-pattern devenv",
```

Comment:
Will the paths in the `.eslintignore` add anything more then whats already in the `.gitignore`?

Reply:
Yeah it sounded good to use the gitignore file but @Clarity-89 raised a concern that because of this the enterprise code is not detected. Because it is added to the gitignore.

Reply:
Aha, good catch! 👍